### DISCUSSION TITLE
Using enter key doesn't respect "hold-to-confirm" when doing dangerous operation.

### DISCUSSION THREAD

**Discussion by IkaWaAyuMu** [ppy/osu#25932] [Ideas]
### Type

Game behaviour

### Bug description

Using enter doesn't respect "hold-to-confirm" when doing dangerous operation such as delete a beatmap, disable "featured artists" filter etc.

### Screenshots or videos

https://github.com/ppy/osu/assets/7066150/52a8f916-8834-43b1-814b-8965474ff2d4

### Version

2023.1220.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/13721350/compressed-logs.zip)

**Comment by bdach** [comment]
The number hotkey also doesn't wherever supported.

I dunno, it was a half "feature".

**Comment by peppy** [comment]
I'm not sure it should. The behaviour was intentionally implemented as it is, but I would be up for reconsidering if there's enough feedback.

**Comment by IkaWaAyuMu** [comment]
I thought it's not intentional when I reported those. But I agree that it can left like that as we can't do dangerous operation using only keyboard (ex. You can't turn off FA filter using shortcut not quick delete beatmaps using Del+Enter).

**Comment by SollyBunny** [comment]
I don't think turning off featured artists should be considered a dangerous operation, specially when you're trying to download a beatmap really quickly it's quite annoying

**Reply by IkaWaAyuMu** [reply]
This discussion is related to operation behavior, not what to be considered "dangerous operation". So, if you want to further discuss on this, I recommended to create new discussion.

---
For your question
If I recalled correctly, Turn off "Featured Artist" is considered dangerous operation because they want to encourage licensed content, and may be easier to deal with software publisher such as "store" - Think of why "Cyt\*\*d", "Ast\*\*DX", or other community-made-content-based game is very hard to download, setup on mobile - and osu! is still struggle with this also.
For context ->  https://github.com/ppy/osu/pull/21646
